build/dependencies.props

```diff
@@ -1,5 +1,6 @@
 <Project>
   <PropertyGroup>
+    <CachingVersion>2.1.1</CachingVersion>
```

Comment:
Dependency on [Microsoft.Extensions.Caching.Memory/2.1.1](https://www.nuget.org/packages/Microsoft.Extensions.Caching.Memory/2.1.1) will require users on .net >= 4.6.1 to consume .netstandard2.0 metapackage. In some cases this could cause conflicts.
It might be worthwhile exploring if it would be better to take a dependency on [5.0.0](https://www.nuget.org/packages/Microsoft.Extensions.Caching.Memory/5.0.0) as it has a dedicated .net461 target.

Reply:
Oh, that's a good point. I've switched the dependency over to 5.0.0 and there doesn't seem to be any issues with it.